source/users/custom-metadata.txt

```diff
@@ -15,6 +15,17 @@ Enable Custom User Data
 Overview
 --------
 
+You can define :ref:`custom user data <custom-user-data>` in MongoDB Atlas and 
+configure App Services to store and access each user's data. App Services can also read 
+:ref:`user metadata <configure-user-metadata-on-the-backend>` from authentication 
+providers. You can configure App Services to request and access the metadata that you 
+want from each provider.
```

Comment:
```suggestion
providers.
```

removing b/c this is more detail than needed here. this is info better suited for the below user metadata section.